specification/ml/_types/TrainedModel.ts

```diff
@@ -429,9 +429,9 @@ export class TrainedModelPrefixStrings {
   /**
    * String prepended to input at ingest
    */
-  ingest: string
+  ingest?: string
```

Comment:
The ingest and search params are optional, you can set one or the other or both or neither